sdk/servicebus/service-bus/src/diagnostics/tracing.ts

```diff
@@ -22,12 +27,16 @@ export const tracingClient = createTracingClient({
  */
 export function toSpanOptions(
   serviceBusConfig: Pick<ConnectionConfig, "host"> & { entityPath: string },
-  spanKind?: TracingSpanKind
+  operation: MessagingOperationNames,
+  spanKind?: TracingSpanKind,
 ): TracingSpanOptions {
   const spanOptions: TracingSpanOptions = {
     spanAttributes: {
-      "message_bus.destination": serviceBusConfig.entityPath,
-      "peer.address": serviceBusConfig.host,
+      "hostname": serviceBusConfig.host,
+      "messaging.system": "servicebus",
+      "entity-path": serviceBusConfig.entityPath,
```

Comment:
for publish operation:
```suggestion
      "messaging.destination.name": serviceBusConfig.entityPath,
```

for receive/process/settle operations:
```suggestion
      "messaging.source.name": serviceBusConfig.entityPath,
```


Note: entityPath should be the name without partition/subscription